common/src/nexus_config.rs

```diff
@@ -285,6 +286,7 @@ pub struct PackageConfig {
     #[serde(default)]
     pub tunables: Tunables,
     /// `Dendrite` dataplane daemon configuration
+    #[serde(default)]
     pub dendrite: DpdConfig,
```

Comment:
With this PR, is this still required?

Reply:
Hmmm, I'm going to lean towards yes, it will be okay to resolve dendrite via internal DNS going forward, and yes, it will _probably_ break the integration tests if we remove `DpdConfig` right now. Hard for me to say for sure since I'm not up to speed on how DNS records will be created in the testing context. In #2694 we are creating a `dpd` instance per test like we do with cockroachdb and clickhouse.